.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Nova Funcionalidade/New Feature
+about: Eu quero sugerir uma nova funcionalidade para o app/I want to suggest a new feature
+title: ''
+labels: ''
```

Comment:
Couldn't we also add the related tags here?

Reply:
Same above

Reply:
Same as above